Documentation/2.0/admin_guide.md

```diff
@@ -162,20 +162,21 @@ etcd sets different timeouts for various types of client requests. The timeout v
 
 #### Get requests
 
-No timeout is set for get requests, because etcd can always return the result in a short time.
+Timeout is not set for get requests, because etcd serves the result locally in a short time.
```

Comment:
The main reason is that it is also non-blocking. For example, zk block read even though it serves locally to ensure local ordering.